leaderboard/templates/leaderboard/competition.html

```diff
@@ -92,12 +96,12 @@ <h2>Leaderboard of {{competition.name}}</h2>
         <h3>{{testset}}</h3>
         <div class="table-responsive">
           <table class="table table-striped table-sm">
-            <thead>
+            <thead class="thead-dark">
               <tr>
                 <th>#</th>
                 <th>Name</th>
-                <th>SacreBLEU score</th>
-                <th>chrF score</th>
+                <th>BLEU</th>
```

Comment:
@snukky: why drop `SacreBLEU` and refer to `BLEU` only?

Reply:
Yes, for brevity. Table sorting adds extra elements to `<thead>`s.